graphql/tests/mutation/addAnnotationIssue/data.json

```diff
@@ -1 +1,8 @@
-{}
+{
```

Comment:
I don't see the result files to complete these tests. Did you forget to push them?

Reply:
This was just added to ensure that `task-1234` exists at the time of doing [this](https://gist.github.com/hadjri/b6cf1c8b37c84a932cdeb6f4313d87f8) mutation, or else the graphql test would error.